cloud-formation/cfn.yaml

```diff
@@ -282,3 +282,29 @@ Resources:
     DependsOn:
     - TargetGroup
     - ElasticLoadBalancer
+
+  StateMachineUnavailableMetric:
+    Type: "AWS::Logs::MetricFilter"
+    Properties:
+      LogGroupName: !Sub support-frontend-${Stage}
+    FilterPattern: "Regular contributions state machine is not available"
+    MetricTransformations:
+     MetricNamespace: !Sub support-frontend-${Stage}
+      MetricName: state-machine-unavailable
+      MetricValue: 1
+
+  StateMachineUnavailableAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
```

Comment:
I would suggest applying the CreateProdResources condition once you're happy with the testing of this.